airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -144,11 +149,14 @@ def _fetch_dag_runs(
         start_date_gte,
         start_date_lte,
     )
+
+    query = apply_array_filter(query, key=DagRun.state, values=state)
+
     # Count items
     total_entries = query.count()
     # sort
     to_replace = {"dag_run_id": "run_id"}
-    allowed_filter_attrs = [
+    allowed_sort_attrs = [
```

Comment:
Renamed as the previous name was slightly misleading, as it was not filtering anything.

Reply:
Likely a copy-paste error, thanks for cleaning this up.